mercat/Cargo.toml

```diff
@@ -23,16 +23,12 @@ byteorder = { version = "^1.2.3", default-features = false, features = ["i128"]
 
 # Crypto
 sha3 = { version = "0.8", default-features = false }
-cryptography_core = { path = "../cryptography-core" }
+cryptography_core = { path = "../cryptography-core", default-features = false }
```

Comment:
@fmiguelgarcia so do you think this was the root cause?

Reply:
Yes, I do. I also exported some symbols in order to simplify how to use the library, and clean some deps.